src/main/java/responseCodes/codes/StatusCode.java

```diff
@@ -0,0 +1,5 @@
+package responseCodes.codes;
```

Comment:
According to java code convention Package name cannot be CamelCase. **responseCodes**